packages/components/src/Select/Select.mdx

```diff
@@ -27,12 +28,13 @@ to choose from.
 ## Sizes
 
 <Playground>
-  <Select size="small">
-    <Option value="">Small</Option>
-    <Option value="sad" disabled>Tony</Option>
-    <Option value="old">Steve</Option>
-    <Option value="splat" label="Natasha"></Option>
-  </Select>
+  <Content>
```

Comment:
Why does this have a content wrapper now?

Reply:
because all of the examples are in one playground. i'd probably just separate them